MREUnityRuntime/MREUnityRuntimeLib/App/MixedRealityExtensionApp.cs

```diff
@@ -783,6 +780,33 @@ bool ancestorInList(GameObject go)
 			}
 		}
 
+		private Guid GenerateObfuscatedUserId(IHostAppUser hostAppUser)
+		{
+			if (GrantedPermissions.HasFlag(Permissions.UserTracking) && GlobalAppId != string.Empty)
+			{
+				using (SHA256 hasher = SHA256.Create())
+				{
+					var encoder = new UTF8Encoding();
+
+					StringBuilder idString = new StringBuilder();
```

Comment:
If it's just the one interpolated string, why use a string builder?

Reply:
This was originally appended to later when I was working through this implementation.  Can just be a string now and I will update it to be.

---
In reply to: [453088797](https://github.com/microsoft/mixed-reality-extension-unity/pull/216#discussion_r453088797) [](ancestors = 453088797)